### PR TITLE
Landing page and Login/Signup

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

I have added a landing page and login/signup feature to my app.

Demo Link : https://water-notes-5jx5k756o-shlok30.vercel.app/

Video Demo : 

https://user-images.githubusercontent.com/60795588/176996223-0102fea6-d1d7-4eb2-9a07-286cdea71a95.mov

Can you please review it?

Thanks

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,15 +6,19 @@ import Homepage from "./pages/homepage";
...
     <>
       <Nav />
-      <div className="flex m5-top">
+      <div className="flex space-between m5-top">
         <Sidebar />
         <Routes>
+          <Route path = "/" element = {<LandingPage />} />
           <Route path = "/login" element = {<LoginPage />} />
+          <Route path = "/signup" element = {<SignUp />} />
           <Route path = "/homepage" element = {<Homepage />} />
```
Comment (SRUJANA-PENUGONDA13) [inline src/App.js:22]: Try to add private routes as well

File: src/context/auth-context.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { createContext, useContext, useState } from "react";
+
+const AuthContext = createContext()
+
+const AuthContextProvider = ({children}) => {
+
+    const [email, setEmail] = useState("")
```
Comment (SRUJANA-PENUGONDA13) [inline src/context/auth-context.js:7]: Why are you keeping these details in a context. I don't see these can be reused in anywhere else. 

Ex- Confirm password only during signup only. you are using it and validating after that there is no need of it right.

**Review by SRUJANA-PENUGONDA13 — COMMENTED** [review]
Login and signup pages looks good. I have added few suggestions can you please go through it once.

File: src/App.js (hunk 1 above)
Comment (shlok30) [reply src/App.js:22]: Will add it in the next branch

File: src/context/auth-context.js (hunk 2 above)
Comment (shlok30) [reply src/context/auth-context.js:7]: I'm sharing email and password between login and signup pages
I could've initialised the state in app.js as well but I wanted to keep my state and updates centralised in a context

**Review by TejasShekar — APPROVED** [review]
Looks good. All functionality working as intended to. Make them responsive as well.

**Pull request merged**